themes/bootstrap3/templates/admin/menu.phtml

```diff
@@ -4,4 +4,5 @@
   <a href="<?=$this->url('admin/config')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == "config" ? ' active' : ''?>"><?=$this->transEsc('Configuration')?></a>
   <a href="<?=$this->url('admin/maintenance')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == "maintenance" ? ' active' : ''?>"><?=$this->transEsc('System Maintenance')?></a>
   <a href="<?=$this->url('admin/tags')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == "tags" ? ' active' : ''?>"><?=$this->transEsc('Tag Maintenance')?></a>
+  <a href="<?=$this->url('admin/overdrive')?>" class="list-group-item<?=strtolower($this->layout()->templateName) == "overdrive" ? ' active' : ''?>"><?=$this->transEsc('Overdrive API')?></a>
```

Comment:
Should we display this conditionally based on whether Overdrive is configured?

Reply:
Yes.  I think you showed me how to do this and I added it.  Maybe it was code that somehow got reverted.  I'll go back through my notes and try to recreate it.

Reply:
You did something similar in the MyResearch menu; that might be what you're thinking of. Not sure if the same rule applies here or if a different approach is needed.

Reply:
I added another configuration for this menu to Overdrive.ini as I did for the myResearch menu.  There isn't really a global on/off switch for the integration.  If there is a record type that is overdrive then it's on.